src/main/java/com/gmail/inverseconduit/javadoc/ClassInfoXmlParser.java

```diff
@@ -0,0 +1,179 @@
+package com.gmail.inverseconduit.javadoc;
+
+import java.util.Arrays;
+
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+
+import com.gmail.inverseconduit.utils.DocumentWrapper;
+
+/**
+ * Parses {@link ClassInfo} objects from XML documents.
+ * @author Michael Angstadt
+ */
+public class ClassInfoXmlParser {
```

Comment:
That's a good way of thinking about it.  The Javadocs should not describe "how" the code does what it does (i.e. the implementation details).  They should describe the inputs, outputs, etc.